src/test/java/uk/gov/laa/gpfd/config/builders/HttpSecuritySessionManagementConfigurerBuilderLocalTest.java

```diff
@@ -51,4 +51,19 @@ void shouldLoadPageIfValidSession() throws Exception {
                 .andExpect(status().isOk()).andExpect(jsonPath("$.id").value(reportId));
     }
 
+    //TODO: a test that confirms h2-console can be accessed? Can't get that test to work currently but would be nice
```

Comment:
Can we raise a ticket to do this and put in backlog instead of leaving a comment. We'll never get round to it and this will live in code forever 😆